bonsai-core/src/main/scala/com/stripe/bonsai/IndexedBitSet.scala

```diff
@@ -0,0 +1,301 @@
+package com.stripe.bonsai
+
+import scala.annotation.switch
+import scala.collection.immutable.BitSet
+import scala.collection.mutable.{ Builder, ArrayBuilder }
+
+/**
+ * IndexedBitSet is an immutable set for storing non-negative integer
```

Comment:
super nit, feel free to ignore, but... is it worth mentioning that it is, for all reasonable purposes, read-only?